lib/phlex/context.rb

```diff
@@ -24,12 +28,21 @@ def component(component, *args, **kwargs, &block)
       end
 
       if args.one? && !block_given?
-        component.new(**kwargs) { text(args.first) }.call(target.children)
+        component.new(**kwargs) { text(args.first) }.call(@_target)
       else
-        component.new(*args, **kwargs, &block).call(target.children)
+        component.new(*args, **kwargs, &block).call(@_target)
       end
     end
 
+    def _render_block(new_target, ...)
+      old_target = @_target
+      @_target = new_target
```

Comment:
Do we still need those? Now `@_target` is always going to be the same buffer right?

Reply:
You’re right, we don't need it.